ui/component/livestreamCompositor/style.scss

```
@@ -106,6 +106,63 @@
+  color: #fff;
+  font-weight: 700;
+  font-size: 0.7em;
+  flex: 0 0 auto;
+}
+
+.livestream-compositor__chat-edit-hyperchat-body {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4px;
+  padding: 2px 8px;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)